exercise-1/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  background-color: #3da2e0;
+}
+div {
+  background-color: white;
+  height: 620px;
+  width: 620px;
+  border-radius: 10px;
+  margin: 44px 0px 0px 360px;
+}
+.name {
+  padding: 40px 118px 10px 13px;
```

Comment:
I checked the boxes. They look very different. Please don't give unnecessary padding/margins here and there. Try to understand the design a little more.